source/features/sticky-comment-header.tsx

```diff
@@ -0,0 +1,14 @@
+import './sticky-comment-header.css';
+
+import features from '../feature-manager.js';
+
+void features.addCssFeature(import.meta.url);
```

Comment:
> > when I have commented on someone else's original issue, the sticky header is still fully transparent
> 
> Yeah, happens sometimes because of #6554

Since this feature is CSS, it's not affected by #6554. The perks of CSS-only features 🙌